packages/webcomponents/src/api/Payment.ts

```diff
@@ -148,24 +149,32 @@ export interface IPayment {
   amount_disputed: number;
   amount_refundable: number;
   amount_refunded: number;
+  amount_returned?: number;
   balance: number;
   captured: boolean;
   capture_strategy: CaptureStrategy;
   currency: 'usd';
   description: string;
   disputed: boolean;
-  disputes: IDispute[];
+  disputes: (IDispute | null)[];
```

Comment:
Is it possible to type `disputes` this way? It reads to me like `disputes` will always be a truthy array with either `IDispute` or `null` as items in the array. I know its not possible to have `null` index items in the array

Could it be: `IDispute[] | null` ? 🤔

Reply:
Oh good catch, this is not needed. I changed that during development to test something and forgot to undo.
Thanks!